test/data/v2/pre_and_post/ee.yml

```diff
@@ -0,0 +1,20 @@
+---
+version: 2
+
+additional_build_steps:
+  prepend_base:
```

Comment:
maybe include a mix of scalar and list-typed values in the test to ensure the multi-line splits and coercion are behaving?

Reply:
👍🏻